src/main/java/com/puppycrawl/tools/checkstyle/Main.java

```diff
@@ -724,8 +724,8 @@ private static final class CliOptions {
         /** Option that controls whether to print the AST of the file. */
         @Option(names = {"-t", "--tree"},
                 description = "Prints Abstract Syntax Tree(AST) of the checked file. The option "
-                        + "cannot be used other options and requires exactly one file to run on "
-                        + "to be specified.")
+                        + "cannot be used with other options and requires exactly one file to "
+                        + "run on to be specified with the path.")
```

Comment:
@romani , Please confirm if this is what you meant by

Reply:
>with the path.

Remove this. If all in same folder, no path is required.

Reply:
@romani , I have removed like you said.

> and requires exactly one file to run on to be specified

This still sounds like we are asking just to mention the filename. And yes, if this file is in the same directory, our jar runs. But if the test file is of a project with multiple sub dirs like ours, this description is not entirely helping, especially for new contributors, I believe.

> We have been receiving many queries on our matrix forum from new contributors struggling to use our jar through CLI.
> The problem here almost in every case was the download path and cli launch path being different (or) the test file/config not located at the same directory as jar.

We have resolved the problem of cli launch path with the other line that I added but the second problem I'm not able to find the right wording. Please tell me if we leave it like this or if you have any nice wording to add :)